# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../disktype/dont-use-fs.h.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./file.c.orig	2004-02-10 22:14:33.269214616 +0000
+++ ./file.c	2004-02-10 22:14:41.536957728 +0000
@@ -32,7 +32,7 @@
 
 #ifdef USE_IOCTL_LINUX
 #include <sys/ioctl.h>
-#include <linux/fs.h>
+#define BLKGETSIZE _IO(0x12,96) /* return device size /512 (long *arg) */
 #endif
 
 #ifdef USE_IOCTL_FREEBSD
